From ad2637520e54d49c886f4c086249c2dfafdb8b70 Mon Sep 17 00:00:00 2001
From: Mandy Lavi <mandy.lavi@freescale.com>
Date: Tue, 16 Sep 2014 14:49:55 +0300
Subject: [PATCH 025/399] fmd: fix casting warning

Coverity issue

Change-Id: If0a7e1dffa09e3513b7ccb13a1dc09bf91056238
Signed-off-by: Mandy Lavi <mandy.lavi@freescale.com>
Reviewed-on: http://git.am.freescale.net:8181/18831
Reviewed-by: Mandy Lavi <Mandy.Lavi@freescale.com>
Tested-by: Mandy Lavi <Mandy.Lavi@freescale.com>
Reviewed-by: Madalin-Cristian Bucur <madalin.bucur@freescale.com>
[Lu:Original patch taken from
QorIQ-SDK-V1.7-SOURCE-20141218-yocto.iso]
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 .../freescale/fman/src/wrapper/lnxwrp_sysfs_fm.h   |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm.h b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm.h
index e6bf83e..137653e 100644
--- a/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm.h
+++ b/drivers/net/ethernet/freescale/fman/src/wrapper/lnxwrp_sysfs_fm.h
@@ -126,7 +126,7 @@ int fm_profile_dump_regs(void *h_pcd, int ppnum, char *buf, int nn);
 		int k, m = n; \
 		phys_addr_t pa = virt_to_phys(&((st)->phrase)); \
 		k = snprintf(&buf[m], PAGE_SIZE - m, \
-		"0x%010llX: 0x%08x%8s\t%s\n", pa, \
+		"0x%010llX: 0x%08x%8s\t%s\n", (unsigned long long) pa, \
 		ioread32be((uint32_t *)&((st)->phrase)), "", #phrase); \
 		if (k < 0 || m > PAGE_SIZE - 90) \
 			FM_DMP_PGSZ_ERR \
-- 
1.7.5.4

